mitmproxy/tools/web/webaddons.py

```diff
@@ -0,0 +1,18 @@
+class WebOptions:
```

Comment:
This is to stay consistent with https://github.com/mitmproxy/mitmproxy/blob/master/mitmproxy/tools/console/consoleaddons.py.